website/style.css

```diff
@@ -1,11 +1,13 @@
+@import url('https://fonts.googleapis.com/css2?family=PT+Sans&family=Source+Code+Pro&display=swap');
```

Comment:
Can you please provide the used font files in the repo. It's not GDPR-compliant to use external google URLs

Reply:
Ohhh, wait me a second, im downloading the stuff